pkgs/servers/rainloop/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, dataPath ? "/etc/rainloop",
+  edition ? "", hash ? "bf558752ce5035dc036702878f453fa4ba3da68c25e77271a079ac285d8acf76" }: stdenv.mkDerivation rec {
```

Comment:
I don't think we need to expose `edition` or `hash`.

It is easy to enough to override them with:

```
rainloop.overrideAttrs (old: {
  src = fetchurl {
    url = "https://github.com/RainLoop/rainloop-webmail/releases/download/v${version}/rainloop-professional${version}.zip";
    sha256 = "<new-hash>";
  };
})
```